Sources/AsyncHTTPClient/NIOTransportServices/TLSConfiguration.swift

```diff
@@ -64,18 +64,22 @@
             let options = NWProtocolTLS.Options()
 
             // minimum TLS protocol
-            if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *) {
+            if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, macCatalyst 13.0, *) {
```

Comment:
not sure if macCatalyst should be explicit, its implied and, as of now, there is compilation warning:

`Unnecessary check for 'macCatalyst'; enclosing scope ensures guard will always be true`